### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-General_Keyboard: Keyboard focus is not going to the menu list items of the left sidebar and hence those are not accessible using keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#153]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS2.1.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Server" in the left menu section and Press Enter.
5. Verify that menu list items of the left sidebar are accessible using keyboard.

**Actual Result:**
Keyboard focus is not going to the menu list items of the left sidebar and hence those are not accessible using keyboard.

**Expected Result:**
Keyboard focus should go to all the menu list items of the left sidebar and on selecting it using enter key, user should be able to navigate to all the pages of the application.

**Note:** This issue is observed throughout the application.

**User Impact:**
Keyboard only users will be unable to navigate to maximum section or pages of the application.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment**:
[Bug 153_Sidebar menu items are not accessible using keyboard.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331157/Bug.153_Sidebar.menu.items.are.not.accessible.using.keyboard.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
